src/q7/concurrent-INGI2143/exam/exam.mk

```diff
@@ -0,0 +1,3 @@
+TYPE=exam
+BASE_DIR=../../../
+include $(BASE_DIR)../concurrent-INGI2143.mk
```

Comment:
@MartinBraquet @Jimvy I can never remember, what is the consensus with `$(BASE_DIR)`?

Reply:
Here it is needed.
The problem was with the `SOL=`.